src/frontend/contexts/userContext.js

```diff
@@ -26,6 +26,14 @@ const UserProvider = ({children}) => {
                 return { ...state, foundUser : action.payload };
             case 'REMOVEFROMWATCHLATER': 
                 return { ...state, foundUser : action.payload };
+            case 'ADDPLAYLIST': 
+                return { ...state, foundUser : action.payload };
+            case 'REMOVEPLAYLIST': 
+                return { ...state, foundUser : action.payload };
+            case 'ADDVIDEOTOPLAYLIST': 
+                return { ...state, foundUser : action.payload };
+            case 'REMOVEVIDEOFROMPLAYLIST': 
+                return { ...state, foundUser : action.payload };
```

Comment:
Seems like the functionality is being repeated everywhere. You can combine it all together. Like below:

```case 'REMOVEPLAYLIST':   
 case 'ADDVIDEOTOPLAYLIST':   
 case 'REMOVEVIDEOFROMPLAYLIST':   
     return { ...state, foundUser : action.payload };`

Reply:
yes Om , yes would do this this asap